examples/platform_channel/lib/main.dart

```diff
@@ -86,5 +86,12 @@ class _PlatformChannelState extends State<PlatformChannel> {
 }
 
 void main() {
-  runApp(const MaterialApp(home: PlatformChannel()));
+  runApp(MaterialApp(
+    title: 'Flutter platform_channel',
+    theme: ThemeData(
+      primarySwatch: Colors.grey,
+      useMaterial3: true,
+    ),
+    home: const PlatformChannel()
```

Comment:
nit: trailing comma for consistency: 

```suggestion
    home: const PlatformChannel(),
```